src/hb-ot-layout-private.hh

```diff
@@ -375,7 +380,7 @@ _hb_glyph_info_is_default_ignorable (const hb_glyph_info_t *info)
 static inline hb_bool_t
 _hb_glyph_info_is_default_ignorable_and_not_fvs (const hb_glyph_info_t *info)
```

Comment:
Should we update the name of this function to add "_or_tag"? Or maybe use an alternative such as _hb_glyph_info_is_ignorable_for_skipping_iterator?